sled-agent/src/services.rs

```diff
@@ -1376,6 +1376,25 @@ impl ServiceManager {
             .add_instance(ServiceInstanceBuilder::new("default")))
     }
 
+    async fn zone_network_setup_install(
```

Comment:
Is there any reason this needs to be an async function?

Reply:
Not really, I'll change that